openstack/backup-etcd/templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: backup-etcd
+  namespace: backup
+spec:
+  replicas: 1
```

Comment:
What happens if there is a helm deployment during a backup run? The pod will be killed, does this leaves strange artifacts or unfinished swift uplaods? Do we need a rollingUpdate strategy with `maxUnavailable: 0` or so?